app/publish-update-retire-content/organisations-people/organisations.md

```diff
@@ -0,0 +1,402 @@
+
+Follow the [guidance on adding translations to people pages](/publish-update-retire-content/organisations-people/people-roles/#add-a-translation-to-a-role-or-people-page). Once you've added a translated version, it will show up on the translated organisation page automatically. 
+
+## Add or change the accessible formats request email
+
+You need to include an email address so the public can order attachments in alternative formats.
+
+1. On your organisation page, go to the 'Details' tab and select 'Edit'.
+2. Under the heading 'Accessible formats request email', add the email address you want to use.
+3. Select 'Save' at the bottom of the page.
+
+## Add or edit the default news image
+
+You need to include a default image for your organisation that will appear if an editor publishes a news story without an image.
+
+1. On your organisation page, go to the 'Details' tab and select 'Edit'.
+2. Under the heading 'Default news image', select the 'Choose file' button and upload the image from the folder it's saved in.
+3. Select 'Save' at the bottom of the page.
+
+If there's already an image there, it'll replace it with your new image after you select save.  
+
+## Add or update the jobs information
+
+You can let users know how to apply for jobs at your organisation by adding a link in the 'Recruitment URL' section. For example, you can link to the Civil Service Jobs website.
+
+The link will appear as 'Jobs' under the 'Corporate information' section of your organisation page.
+
+1. On your organisation page, go to the 'Details' tab and select 'Edit'.
+2. Under the heading 'Recruitment URL', add the link you want to use.
+3. Select 'Save' at the bottom of the page.
+
+## Closing an organisation page
+
+Organisations should never be deleted from GOV.UK. Instead, the status of an organisation can be changed to 'closed'.
+
+When closed, organisations are not listed on [the organisations index](https://www.gov.uk/government/organisations) and disappear from the parent or sponsor department's organisation page. But they continue to have a profile page findable at the same URL, with a statement indicating they're closed.
+
+Closed organisation profiles will remain findable by external search engines and the internal site search, and remain listed as a filter option, for example when filtering by organisation on the list of all government publications.
+
+### How to close an organisation
+
+1. Go to the 'Details' tab and select 'Edit'.
+2. Scroll to 'Status on GOV.UK' and change it from 'Currently live' to 'Closed'.
+3. Select the appropriate reason for closure.
+4. Scroll to the bottom of the page and select 'Save'.
+
+If your organisation is being taken over by another organisation, enter the name of the organisation taking over from (superseding) your organisation.
+
+The different options for describing 'closed' organisations each generate slightly different wording for users.
+
+| Reason for closure | Use when | Automated wording | Example |
```

Comment:
This table is broken, probably because of the use of markdown heading rows, where as it probably needs to be a Design System HTML table